ebi_eva_common_pyutils/variation/contig_utils.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2020 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+import requests
+from retry import retry
+
+
```

Comment:
Python re-write for the original implementation in "https://github.com/EBIvariation/eva-web/blob/126f7c7324de6a57c8e399f5c6fb2443faff9f94/src/js/views/eva-variant-view.js#L47"